Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM registry.access.redhat.com/ubi8/ruby-27
 
 USER root
 RUN rpm -Uvh https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm \
-  && dnf install --setopt=skip_missing_names_on_install=False,tsflags=nodocs --skip-broken -y shared-mime-info postgresql13 postgresql13-devel postgresql13-libs \
+  && dnf install --setopt=skip_missing_names_on_install=False,tsflags=nodocs --skip-broken -y shared-mime-info postgresql13 postgresql13-libs \
```

Comment:
don't we need the `devel` package anymore?

Reply:
It seems that it is not needed :shrug: 
We do need the others, because we need postgres clients, which is provided by `postgresql13`, and `postgresql13-libs` is its dependency.

Reply:
`-devel` packages are required sometimes when you run `bundle install` locally, for instance, `mysql2` gem requires having `mysql-devel`. I tried to build the image locally and it succeeded, so it seems Postgres gem doesn't need the headers. It's fine for me then.